api_app/api/routes/workspaces.py

```diff
@@ -234,3 +234,10 @@ async def delete_user_resource(workspace_id: str, service_id: str, resource_id:
     await send_uninstall_message(user_resource, user_resource_repo, previous_deletion_status, ResourceType.UserResource)
 
     return UserResourceIdInResponse(resourceId=user_resource.id)
+
+
+@user_resources_router.patch("/workspaces/{workspace_id}/workspace-services/{service_id}/user-resources/{resource_id}", response_model=UserResourceInResponse, name=strings.API_UPDATE_USER_RESOURCE)
+async def patch_user_resource(workspace_id: str, service_id: str, resource_id: str, user_resource_patch: UserResourcePatchEnabled, user_resource=Depends(get_user_resource_by_id_from_path), user_resource_repo=Depends(get_repository(UserResourceRepository)), user=Depends(get_current_user), workspace_service=Depends(get_deployed_workspace_service_by_id_from_path), workspace=Depends(get_workspace_by_workspace_id_from_path)) -> UserResourceInResponse:
```

Comment:
why is this bringing down the workspace service, doesn't seem like it is used here? - even if it is used, it shouldn't need to be deployed

Reply:
a) you should provide the service id as part of the URI.
b) It won't make much sense to deploy a user resource for a service for a service which is not yet deployed. Hence the sanity check...

Reply:
a) once we have the PR in flight that checks the whole user resource id chain - this will already be in effect - so no need for the extra check
b) agree if we would be deploying the resource - but here we are patching the resource -- let's talk tomorrow, but this is an unnecessary DB call, and in some cases we may want to patch and disable even if the service is not deployed

Reply:
talked f2f. agreed to remove the check